src/eko/scale_variations/expanded.py

```diff
@@ -1,5 +1,5 @@
 # -*- coding: utf-8 -*-
-r"""This module contains the scale variation operator for the expanded scheme (``ModSV=expanded``).
+r"""Contains the scale variation operator for the expanded scheme (``ModSV=expanded``).
```

Comment:
Strange, it should have complained also for the final 's':
```suggestion
r"""Contain the scale variation operator for the expanded scheme (``ModSV=expanded``).
```

Reply:
Isn't it?

Reply:
no, pydocstyle was fine - for this and the commit below ... isn't it for you?

Reply:
I was going by heart: I remember that it was telling to always use imperative style. As you did for functions' docstrings.

I wonder if not complaining about this one is done on purpose, or they are just different, and they forgot to implement the warning for module level.